.prettierrc

```diff
@@ -4,7 +4,7 @@
 	"trailingComma": "es5",
 	"useTabs": true,
 	"printWidth": 80,
-	"endOfLine": "lf",
+	"endOfLine": "auto",
```

Comment:
I suggest we keep this as `lf` and add a matching _.gitattributes_ file with `lf` like we do elsewhere - just makes it more consistent cross platform, especially now that we use WSL more